src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/DefaultRazorSemanticTokenInfoServiceTest.cs

```diff
@@ -18,8 +18,8 @@ public void GetSemanticTokens_NoAttributes()
         {
             var txt = $"@addTagHelper *, TestAssembly{Environment.NewLine}<test1></test1>";
             var expectedData = new List<uint> {
-                1, 1, 5, 0, 0, //line, character pos, length, tokenType, modifier
-                0, 8, 5, 0, 0
+                1, 1, 5, 1, 0, //line, character pos, length, tokenType, modifier
```

Comment:
Did this change because of the token type differences? Aka `tagHelperXYZ` -> `class`?

Reply:
Correct. I'll have to revert  these changes when we go back to our more detailed tokenType names.